Задания с 18.02.2022/Task2/Task2/BWT.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+
+public class BWT
+{
+    private static string leftShift(string line, int count)
```

Comment:
```suggestion
    private static string LeftShift(string line, int count)
```
Методы в C# именуются всегда с заглавной.